doc/typeclass.md

```diff
@@ -224,7 +224,7 @@ As an exercise, try defining default instances for other types, such as `List` a
 The Lean standard library contains the definition `inferInstance`. It has type `{α : Type u} → [i : α] → α`,
```

Comment:
Another occurrence of the type. Not sure if we need both.

Reply:
I've removed the one in the code. Well spotted!